cmd/relayfile-cli/cloudauth.go

```
@@ -0,0 +1,311 @@
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, endpoint, bytes.NewReader(body))
+	if err != nil {
+		return agentRelayStoredAuth{}, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("User-Agent", "relayfile-cli/"+relayfileVersion)
+
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return agentRelayStoredAuth{}, fmt.Errorf("refresh the Agent Relay cloud session at %s: %w", endpoint, err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)